app/Http/Controllers/AuthenticationController.php

```diff
@@ -20,25 +20,16 @@ public function register()
     public function check(Request $request)
     {
         $request->validate([
-            'username'=>'required',
-            'password'=>'required'
+            'username' => ['required'],
+            'password' => ['required'],
         ]);
 
-        $userInfo = User::where('username','=',$request->username)->first();
-        
-        
-        if(!$userInfo){
-            return back()->with('fail','We do not recognize your username');
-        }
-        else{
-            if($request->password == $userInfo->password){
-                $request->session()->put('LoggedUser', $userInfo->id);
-                return redirect()->route('dashboard');
-            }
-            else{
-                return back()->with('fail','Incorrect password');
-            }
+        $credentials = $request->only('username', 'password');
+
+        if (Auth::attempt($credentials)) {
+            return redirect()->intended('dashboard');
```

Comment:
Kindly generate a session. Cause session is needed in middleware 'auth'.

Example code:
` $request->session()->regenerate();`